elements/pfe-readtime/src/pfe-readtime.scss

```diff
@@ -0,0 +1,8 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: readtime;
+
+:host {
+  @include pfe-c-typography($type: text, $sizing: md);
```

Comment:
@alazzara  I saw this CSS which was good in the component, but I don't see it on the anbartle sandbox